### PR TITLE
(feat) Patient search UI enhancements

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Improvements to the patient search UI aimed at getting them in line with the available designs and fixing some small quirks.

## Screenshots

> Desktop
Screenshot 2023-03-19 at 1 05 33 AM

Screenshot 2023-03-19 at 1 05 58 AM

> Tablet
Screenshot 2023-03-19 at 1 06 23 AM
Screenshot 2023-03-19 at 1 06 38 AM

### CODE REVIEW

File: packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss [hunk 1]
```diff
@@ -23,13 +25,16 @@
...
+  color: $text-02;
 }
 
 .patientAvatar {
   width: spacing.$spacing-09;
   height: spacing.$spacing-09;
   margin: spacing.$spacing-03 spacing.$spacing-05 spacing.$spacing-03 0;
   border-radius: 1px;
+  font-weight: unset;
+  line-height: unset;
```
Comment (denniskigen) [inline packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss:37]: Unsets some styling from https://github.com/openmrs/openmrs-esm-core/blob/8bf18b44dc900de42b485e85c408bad354ac4f25/packages/apps/esm-primary-navigation-app/src/components/navbar/navbar.scss#L9 that affects the appearance of the text in the patient avatar.

Comment (vasharma05) [reply packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss:37]: Great!

**Comment by vasharma05** [comment]
Hi @denniskigen !
The identifier tags shown in the compact patient banner doesn't work properly. We'll have to fix that too. 
To reproduce, 
1. Create a patient and add SSN identifier to it. The format of the value but be A-<7 digit number>.
2. Add SSN's UUID in the `defaultIdentifierTypes` configurations.

See you won't see the SSN tag.
Should this be fixed in this PR only or in another PR?
CC: @donaldkibet 
Thanks!

**Review by vasharma05 — APPROVED** [review]
Thanks for working on this!

**Comment by denniskigen** [comment]
> Hi @denniskigen ! The identifier tags shown in the compact patient banner doesn't work properly. We'll have to fix that too. To reproduce,
> 
> 1. Create a patient and add SSN identifier to it. The format of the value but be A-<7 digit number>.
> 2. Add SSN's UUID in the `defaultIdentifierTypes` configurations.
> 
> See you won't see the SSN tag. Should this be fixed in this PR only or in another PR? CC: @donaldkibet Thanks!

Great catch. That should probably be handled in a separate PR.

**Pull request merged**